config/samples/httpbin.yaml

```diff
@@ -1,9 +1,12 @@
 apiVersion: tyk.tyk.io/v1alpha1
 kind: ApiDefinition
 metadata:
-  name: httpbin
+  name: httpbin2
 spec:
-  name: httpbin
+  name: httpbin2
+  contextRef: 
+    name: mycontext3
+    namespace: default
   use_keyless: true
```

Comment:
I wouldn't edit this particular sample, as this is the "hello-world" use-case.
Better to provide specific examples, such as here: `https://github.com/TykTechnologies/tyk-operator/tree/master/config/samples/07-operator-context`

Reply:
I modified it for some testing and forgot to undo changes. Reverted back.